### PR TITLE
adds a style obj for configuration of the modal. Can overwrite the wi…

### PR DESCRIPTION
**Pull request by icylabsDev**
…dth and height and use percentages.

### CODE REVIEW

**Comment by icylabsDev** [comment]
If my code is "bad" the idea is good =) so please use it.

**Comment by shibe97** [comment]
I can understand your Idea.
Why don't you apply this idea to 'setSize' method?
If width or height props has '%' character, marginLeft or marginTop uses '%'.
How is it?

**Comment by icylabsDev** [comment]
ok. Is it ok with the algorithm?

**Comment by shibe97** [comment]
It's almost ok.
I think that the default unit should be 'px'.
When only '%' unit comes from the props, marginLeft or marginTop should be changed.

**Comment by icylabsDev** [comment]
Ok, so I'll leave the 'px' has you have done, then overwrite it if it has '%' on the width or/and height.

**Comment by icylabsDev** [comment]
I have left the "examples" with an example of usage. would you like me to take it? or We can update the Doc + the examples and change it after?

File: src/index.js [hunk 1]
```diff
@@ -24,39 +24,44 @@ export default class Modal extends Component {
...
             style[effect].panel.width = this.props.width + 'px';
             style[effect].panel.marginLeft = '-' + this.props.width / 2 + 'px';
+          }
         }
         if(this.props && this.props.height) {
+          if(this.props.height.charAt(this.props.height.length-1) === '%'){
+            //Use Percentage
+            const height = this.props.height.slice(0,-1);
+            style[effect].panel.height = height + '%';
+            style[effect].panel.marginTop = '-' + height / 2 + '%';
```
Comment (shibe97) [inline src/index.js]: This is a screen capture of the example page.


The vertical position is a little bit shifted.
In this case, you had better use 'vh' like this.
```
style[effect].panel.height = height + 'vh';
style[effect].panel.marginTop = '-' + height / 2 + 'vh';
```
Please confirm it.

**Comment by shibe97** [comment]
Thanks for changing example.
After merging this pull request, I'm gonna change docs.

File: src/index.js (hunk 1 above)
Comment (icylabsDev) [inline src/index.js]: ok no problem. Should I update it?

**Comment by icylabsDev** [comment]
I'm kind new in this github stuff, let me know if I need to do something else ^^

**Comment by shibe97** [comment]
@icylabsDev Thanks, I'll merge.

**Pull request merged**